ilastik/applets/dataSelection/dataSelectionGui.py

```diff
@@ -598,32 +597,34 @@ def _configureOpWithInfos(self, roleIndex, startingLane, endingLane, infos):
         # Resize the slot if necessary            
         if len( opTop.DatasetGroup ) < endingLane+1:
             opTop.DatasetGroup.resize( endingLane+1 )
-        
+
         # Configure each subslot
         for laneIndex, info in zip(list(range(startingLane, endingLane+1)), infos):
             try:
                 self.topLevelOperator.DatasetGroup[laneIndex][roleIndex].setValue( info )
-                return True
             except DatasetConstraintError as ex:
                 return_val = [False]
                 # Give the user a chance to fix the problem
                 self.handleDatasetConstraintError(info, info.filePath, ex, roleIndex, laneIndex, return_val)
                 if return_val[0]:
                     # Successfully repaired graph.
-                    return True
+                    continue
```

Comment:
I don't see the purpose of this 'continue', but it doesn't hurt either.

Reply:
it's verbose. I guess that was kind of forgotten the last time, that it should continue here, not return something

Reply:
Yes. It is at the end of the for loop though, and thus redundant, but emphasising this explicitly actually makes sense. Sorry for the hiccup